source/process.adoc

```diff
@@ -224,16 +226,16 @@ A Specification Committee must approve, by Super-majority, the following lifecyc
 * A Platform designation (this Super-majority must include a Super-majority of Strategic Members of the Working Group); and
 * Service Releases.
 
-A ballot is used to seek Specification Committee approval. All Specification Committee ballots must be scheduled to run for a period of no less than one week.
+A ballot is used to seek Specification Committee approval. Unless otherwise stated in this process (or a Working Group-specific derivative of this process), the default period for all Specification Committee ballots is fourteen (14) days. During that time, any member of a Specification Committee may request that the period be extended to thirty (30) days. No ballot period may be shorter than seven (7) days. A Specification Committee may opt to increase the length of the ballot period, but may not--under any circumstances--reduce any review period to fewer than seven (7) days.
```

Comment:
I already posted my views in the ChangeLog section and in the recent email exchange.  We should not make the EFSP more restrictive.  Leave that up to the defined derivatives.

Reply:
I've changed it to a simple seven day default.